docs/programming/kotlin/fundamentals/functions.md

```diff
@@ -0,0 +1,11 @@
+# Functions
+
+## Declaring a Function
```

Comment:
Naming functions is a major aspect. Don't care about correctness but care about composition. See: https://cs61a.org/assets/slides/06-Functional_Abstraction.pdf